Add first tutorial (#71)
massalabs/docu-dev: docs/tutorial/trading-bot.mdx

---
id: trading-bot
sidebar_label: Trading bot
---

# Autonomous Trading Bot

This video is a great example of how to use Massa's [autonomous smart contracts](/docs/learn/autonomous-sc) to build a trading bot that will automatically buy and sell tokens on the [Dusa](https://dusa.io) decentralized exchange (DEX).

> This tutorial goes through all the tooling required to build a decentralized application on the Massa blockchain, from the smart contract to the front-end interface, and is a great way to get started with Massa.

Reviewer: improvement idea
```suggestion
This tutorial cover all the tooling required to build a decentralized application on the Massa blockchain, from the smart contract to the front-end interface, making it a great way to get started with Massa.
```
